docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+
+
+    my_alexnet_submission (main folder)
+        models (subfolder)
+            base_models.py
+            __init__.py
+        __init__.py
+        setup.py
+
+4. Now we will start making the submission package. In your ``my_model_submissions`` folder,
+   create a new folder (that we will eventually zip to submit) called ``my_alexnet_submission``.
+   In that newly created folder, create (yet another) folder called ``models``.
+   You can see we are building the package up as explained above.
+   Your IDE file/project explorer should look something like this below at this point:
+    .. image:: /docs/source/modules/tutorial_screenshots/subfolders.png
+      :width: 600
+
+5. Next, we are going to add the ``setup.py`` file into the ``my_alexnet_submission`` folder.
+   There are a few ways to do this, but the easiest is just to navigate into the
+   ``sample_model_submission`` folder, and you will see a ``setup.py`` file there. We are going
+   to copy that and place it inside of the ``my_alexnet_submission`` folder. You should
+   be able to do this by just right clicking the file, copying, and then pasting inside
+   the correct folder. It is important to paste it inside the
+   ``my_alexnet_submission`` folder, in order for it to be placed in the right spot.
+   Your project should now look similiar to this:
+    .. image:: /docs/source/modules/tutorial_screenshots/setup.png
+      :width: 600
+
+6. After this we will make the ``__init__.py`` file and place it inside the same folder as ``setup.py`` above.
+   This ``__init__.py`` file is basically just a blank Python file that the submission
+   needs in order to run. So, the easiest way to do this is to use your IDE to create
+   a new Python file inside the correct folder: in Pycharm, you can do this by highlighting
+   the ``my_alexnet_submission`` folder by clicking it, and it will be shown in blue.
+   From there::
+    right click -> new -> Python file
+   Name this file ``__init__.py`` and
+   click enter.  In the file, hit a new line (enter) so that the ``__init__.py`` file is not blank.
+   Your package should now look akin to this:
+    .. image:: /docs/source/modules/tutorial_screenshots/init_py.png
+      :width: 600
+7. We are almost done! Copy the ``__init__.py`` file you just made and place that
+   *additional* copy inside the ``models`` folder. Finally, we want to add the
+   actual submission to the package. There are a few ways to do this, but
+   for now we are just going to copy the code from ``pytorch.py`` into a blank
+   Python file. Create a new Python file called ``base_models.py`` (the
+   creation process is identical to how you created the ``__init__.py`` file above,
+   just make sure this file is created inside the models folder), and paste the
+   code from ``pytorch.py`` into there. This creates another instance of the
+   ``base_models.py`` file, filled with ``pytorch.py`` ’s code, which is in this case the
```

Comment:
as above, let's work directly with the `base_models.py` file given by `sample-model-submission`